src/components/Layout/HeaderDropDown.js

```diff
@@ -30,14 +30,19 @@ class HeaderDropDown extends React.Component {
 
   render () {
     return (
-      <div className={cx(this.props.className, style.dropDownItem)}>
+      <a className={cx(this.props.className, style.dropDownItem)}>
         <FontAwesome name={this.props.fontAwesome}
                      className={cx(this.state.dropDownActive ? style.itemActive : '', style.item)}
                      onClick={this.toggleDropDown}/>
         <div className={cx(style.dropDown, this.state.dropDownActive ? style.dropDownActive : '')}>
-          {this.props.children}
+          {
+            React.Children.map(this.props.children,
+              child => React.cloneElement(child, {
+                closeDropDownCallback: this.handleClickOutside
```

Comment:
the way it was done before didn't exist. It just closed because the language callback was executed which made the endpoint refetch data about available languages and delete the DropDown from the header for a short period of time.
Since this is quite common practice I think (about the first google page) no doc needed.